.github/workflows/zxc-e2e-test.yaml

```
@@ -163,7 +163,7 @@ jobs:
         if: ${{ runner.os == 'linux' && (inputs.npm-test-script == 'test-e2e-node-local-build' || inputs.npm-test-script == 'test-e2e-node-add') && !cancelled() && !failure() }}
         run: |
           cd ..
-          git clone https://github.com/hashgraph/hedera-services.git  --branch v0.53.2
+          git clone https://github.com/hashgraph/hedera-services.git  --branch v0.54.0-alpha.4
```

```suggestion
          git clone https://github.com/hashgraph/hedera-services.git  --depth 1 --branch v0.54.0-alpha.4
```
this will skip downloading all of the unnecessary history and just download for the tag